src/command_modules/azure-cli-vm/azure/cli/command_modules/vm/tests/latest/test_vm_commands.py

```diff
@@ -188,14 +188,6 @@ def test_vm_image_list_skus(self):
         result = self.cmd("vm image list-skus --location {loc} -p {pub} --offer {offer} --query \"length([].id.contains(@, '/Publishers/{pub}/ArtifactTypes/VMImage/Offers/{offer}/Skus/'))\"").get_output_in_json()
         self.assertTrue(result > 0)
 
-    @AllowLargeResponse(size_kb=2048)
-    def test_list_skus_contains_zone_info(self):
```

Comment:
To mitigate big recording files, I am merging the test scenarios with `test_list_compute_skus_table_output` below